app/src/main/java/com/courseproject/tindar/ChatActivity.java

```diff
@@ -0,0 +1,35 @@
+package com.courseproject.tindar;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.os.Bundle;
+import android.widget.EditText;
+import android.widget.ImageButton;
+
+public class ChatActivity extends AppCompatActivity {
```

Comment:
I think @yiwxng might be using the name Chat for her features. Can you guys talk to each other to make sure the naming is clear between your feature and her feature?

Reply:
No current direct conflict, but thanks for bringing that up. We've already spoken about it but it's good to have in writing; I've messaged Yi.

Reply:
I agree with that. And will be using key word "convo" for conversation to distinguish between Chat and Conversation